### PR TITLE
sortable amp ad implementation

### PR DESCRIPTION
**Pull request by Rooke**
ITI #2698

### CODE REVIEW

File: ads/sortable.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export function sortable(global, data) {
+  validateDataExists(data, ['site', 'name']);
+
+  const slot = global.document.getElementById('c');
+  const ad = global.document.createElement('div');
+  ad.className = 'ad-tag';
+  ad.setAttribute('data-ad-name', data.name);
+  ad.setAttribute('data-ad-size', data.width + 'x' + data.height);
+  slot.appendChild(ad);
+  loadScript(global, 'https://tags-cdn.deployads.com/a/' + data.site + '.js');
```
Comment (dvoytenko) [inline ads/sortable.js]: Please use `encodeURIComponent` on `data.size`.

File: ads/sortable.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (erwinmombay) [inline ads/sortable.js]: 2016 here and every other new file

**Comment by Rooke** [comment]
I squashed the suggested fixes, perhaps prematurely. Is there any further action for me to take to get this OKed?

**Comment by dvoytenko** [comment]
Thanks, @Rooke ! LGTM. Merging.

**Pull request merged**